doc/source/_static/numpy.css

```diff
@@ -0,0 +1,137 @@
+  @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap');
+
+.main {
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+}
+
+div.top-scipy-org-logo-header {
+  background-color: #fafafa;
+  border-bottom: 2px solid #013243; /* Warm Black */
+  margin-top: 0;
+  box-shadow: none;
+}
+div.top-scipy-org-logo-header img {
+  height: 110px;
+  padding-left: 27px;
+}
+
+div.spc-navbar .nav-pills > li > a {
+  background-color: #4d77cf; /* Han Blue -- actually, no name, a little lighter than hans */
+}
+
+div#spc-section-body h1, h2, h3, h4, h5, h6 {
+    font-family: 'Lato', sans-serif;
+}
+
+div#spc-section-body h1 {
+    color: #004242;
+    font-weight: 700;
+    font-size: 320%;
+    text-align: right;
+    letter-spacing: -2px;
+    word-spacing: .03em;
+    margin-top: 9%;
+    margin-bottom: 10%;

```

Comment:
Using percentages for margins results in very weird behavior when the window is resized - percentages are always a fraction of the width, not the height - so this makes the page stretch down when it is widened!

Reply:
I'd recommend using `em` units, since those scale with the text size not the window width.